css/style.css

```diff
@@ -0,0 +1,968 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+a,
+a:hover {
+  text-decoration: none;
+}
+
+/* color section */
+.c-black {
+  color: #000;
+}
+
+.gray {
+  color: #aaa;
+}
+
+.push_left {
+  padding-left: 106px;
+}
+
+.d_none {
+  display: none;
+}
+
+.black-lt {
+  color: #555;
+}
+
+.white {
+  color: white;
+}
+
+.gray-lt {
+  color: #a5a5a5;
+}
+
+.grid {
+  display: grid;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-col {
+  flex-direction: column;
+}
+
+.aic {
+  align-items: center;
+}
+
+.jcc {
+  justify-content: center;
+}
+
+.jcsb {
+  justify-content: space-between;
+}
+
+.jsb {
+  justify-content: space-between;
+}
+
+.jsa {
+  justify-content: space-around;
+}
+
+.jce {
+  justify-content: end;
+}
+
+.fa_pad {
+  padding-right: 16px;
+}
+
+.font_18 {
+  font-size: 18px;
+}
+
+/* navigation bars start here */
+.top-nav {
+  grid-template-areas: "news news news news news about";
+  height: 2rem;
+  background: black;
+  font-size: 13px;
+}
+
+.nav-item {
+  color: white;
+}
+
+.nav_a {
+  padding: 0 10px;
+  color: white;
+}
+
+.news {
+  grid-area: news;
+  align-items: center;
+  padding-left: 10px;
+}
+
+.about {
+  grid-area: about;
+  justify-content: flex-end;
+  align-items: center;
+  padding-right: 0;
+}
+
+.sticky_wrapper {
+  grid-template-columns: 5fr 2fr;
+  grid-template-rows: repeat(2, 2fr);
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  z-index: 5;
+  height: 4.6875rem;
+}
+
+.sticky_nav_bar {
+  grid-auto-flow: column;
+}
+
+.latest1 {
+  position: relative;
+  grid-column: span 4;
+}
+
+.main {
+  position: relative;
+}
+
+.main > * {
+  display: inline;
+  list-style: none;
+  padding: 18px 12px 0 12px;
+  font-size: 14px;
+  color: #aaa;
+}
+
+.logo_img {
+  /* background-color: #f42; */
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  display: none;
+  height: 2.8125rem;
+  padding: 16px 20px;
+  width: 6.625rem;
+  position: absolute;
+  top: -13px;
+  left: 0;
+}
+
+.logo_display {
+  display: block;
+}
+
+.first_one {
+  padding: 0 12px 0 20px;
+}
+
+.sticky_right {
+  justify-content: space-around;
+  align-items: center;
+}
+
+.right_sticky1 {
+  background: url(/images/search.png) no-repeat;
+  width: 2.8125rem;
```

Comment:
- [ ] The Search Image doesn't show up on GitHub Pages, in order to fix it, you need to add a dot before the first slash, to tell exactly where the image is.